src/Typesense/Configuration.js

```diff
@@ -29,7 +41,7 @@ class Configuration {
   }
 
   _isNodeMissingAnyParameters (node) {
-    return !['protocol', 'host', 'port', 'apiKey'].every((key) => {
+    return !['protocol', 'host', 'port', 'path', 'apiKey'].every((key) => {
```

Comment:
@dpastoor One small thing I just noticed: the `path` need not be mandatory here since are already handling an undefined `path` to maintain backward compatibility.

Reply:
@kishorenc yes, if you check in line 18 above in Configuation, I'm handling this by checking if path was defined, and if not, adding it as an empty string. This way, the path element will always be present, so won't get undefined behavior anywhere if it was not set. Likewise, by the time it does this validation, it will be there

Reply:
👍